projects/coredns/coredns/1-22/patches/0001-Bump-golang-x-net-and-golang-x-text-dependency.patch

```diff
@@ -0,0 +1,298 @@
+From acdae75f6dc6dc44bc25372ebc4c991d16c61e45 Mon Sep 17 00:00:00 2001
+From: Sajia Zafreen <szafreen@amazon.com>
+Date: Tue, 2 May 2023 22:59:34 +0000
+Subject: [PATCH] Bump-golang-x-net-and-golang-x-text-dependency
+
+Signed-off-by: Sajia Zafreen <szafreen@amazon.com>
+---
+ go.mod | 83 +++++++++++++++++++++++++++++++++++++++++++++++++++-----
+ go.sum | 86 +++++++++++-----------------------------------------------
+ 2 files changed, 92 insertions(+), 77 deletions(-)
+
+diff --git a/go.mod b/go.mod
+index a72ab760..38db0091 100644
+--- a/go.mod
++++ b/go.mod
+@@ -1,13 +1,16 @@
+ module github.com/coredns/coredns
+ 
+-go 1.16
++go 1.17
```

Comment:
I don't think this will cause issues because newer versions are backwards compatible.